### PR TITLE
[#168195930] End list component

### PR DESCRIPTION
**Pull request by chiadav**
This PR add a smile to the end of list in messages functionality.

| iOS  | Android |
| ------------- | ------------- |
| ios1  | android2 |
| ios2  | android1 |
| ios3  | android3 |

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168195930](https://www.pivotaltracker.com/story/show/168195930): Come CIT quando arrivo alla fine dello scroll di una lista voglio vedere un margine e una icona

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/screens/EdgeBorderComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  emptyListWrapper: {
+    alignItems: "center",
+    marginTop: 74
```
Comment (Undermaken) [inline ts/components/screens/EdgeBorderComponent.tsx]: What if we add some margin at bottom too?

### without bottom margin


### within bottom margin

Comment (chiadav) [reply ts/components/screens/EdgeBorderComponent.tsx]: Add border in 1c107a6:

AddMargin

Comment (chiadav) [reply ts/components/screens/EdgeBorderComponent.tsx]: @matteodesanti : this is the result:

|iOS                          |Android                      |
|-----------------------------|-----------------------------|
|iOS|Android|

**Comment by Undermaken** [comment]
@chiadav don't forget https://github.com/teamdigitale/io-app/pull/1297#pullrequestreview-288554115 :wink:

**Comment by Undermaken** [comment]
@valentinacala Since I already reviewed this PR, I'm going to complete it.

**Comment by Undermaken** [comment]
Added  end list component too:

### service list (old version)


### profile screen


### new service home 


### services search

**Pull request merged**